packages/@okta/vuepress-site/docs/release-notes/index.md

```diff
@@ -13,7 +13,8 @@ The release notes are organized by year in descending order. In the left-side na
 
 We release first to preview orgs and then production orgs. Dates for preview releases are the earliest possible release date. Always check your org to verify the release for your org.
 
-To verify the current release for an org, check the footer of the Admin Console. If necessary, click **Admin** to navigate to your Admin Console.<br>
+To verify the current release and Okta solution for an org (Identity Engine or Classic), check the footer of the Admin Console. The version number is appended with E for Identity Engine orgs and C for Classic Engine orgs. If necessary, click **Admin** to navigate to your Admin Console. <br>
+
 ![Release Number in Footer](/img/release_notes/version_footer.png)
```

Comment:
I don't know if you want to center the image with the new image classes?
`<div class="half"> ![Release Number in Footer](/img/release_notes/version_footer.png) </div>`
the problem with these classes is that "half" is too small, and "three-quarters" is too big. we need an "orig" for original size ;-)

Reply:
Will center original with our workaround.